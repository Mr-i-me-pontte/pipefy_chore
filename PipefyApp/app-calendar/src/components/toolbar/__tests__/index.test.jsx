import React from 'react';
import renderer from 'react-test-renderer';

import Toolbar from '../';
import props from '../../../../__mocks__/big_calendar';

describe('Toolbar', () => {
  it('render with a truthy loading props', () => {
    const component = renderer.create(<Toolbar {...props} loading />);
    expect(component.toJSON()).toMatchSnapshot();
  });

  it('render with a falsy loading props', () => {
    const component = renderer.create(<Toolbar {...props} loading={false} />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
