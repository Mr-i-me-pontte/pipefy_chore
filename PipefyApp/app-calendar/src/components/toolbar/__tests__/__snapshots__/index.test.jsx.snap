// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toolbar render with a falsy loading props 1`] = `
<div
  className="rbc-toolbar"
>
  <span
    className="rbc-toolbar-label"
  >
    Testing label
  </span>
  <span
    className="rbc-btn-group pipe-navigate"
  >
    <a
      className="prev"
      href="#navigate_prev"
      onClick={[Function]}
      tabIndex="0"
      title="Previous"
    >
      <span
        className="pp-ico-bold-arrow-left"
      />
    </a>
    <a
      className="today"
      href="#navigate_today"
      onClick={[Function]}
      tabIndex="0"
      title="Today"
    >
      Today
    </a>
    <a
      className="next"
      href="#navigate_next"
      onClick={[Function]}
      tabIndex="0"
      title="Next"
    >
      <span
        className="pp-ico-bold-arrow-right"
      />
    </a>
  </span>
  <span
    className="rbc-btn-group"
  >
    <button
      className="rbc-active"
      onClick={[Function]}
      tabIndex="0"
    >
      month
    </button>
    <button
      className=""
      onClick={[Function]}
      tabIndex="0"
    >
      week
    </button>
    <button
      className=""
      onClick={[Function]}
      tabIndex="0"
    >
      day
    </button>
    <button
      className=""
      onClick={[Function]}
      tabIndex="0"
    >
      agenda
    </button>
  </span>
</div>
`;

exports[`Toolbar render with a truthy loading props 1`] = `
<div
  className="rbc-toolbar"
>
  <span
    className="rbc-toolbar-label"
  >
    Testing label
  </span>
  <span
    className="rbc-btn-group pipe-navigate pp-position-absolute"
  >
    <span
      className="pp-ico-loading pp-color-info"
    />
  </span>
</div>
`;
